mathics/eval/plot.py

```diff
@@ -1,13 +1,16 @@
 """
 Evaluation routines for 2D plotting.
+
+Note this is distinct from formatting/rendering, e.g. to SVG..
```

Comment:
Maybe "These routines build Expressions that describe plots as Graphics in WL. Note this is distinct from formatting/rendering e.g. to SVG."

Reply:
Ok - I had changed this in the interim (also thinking it might be deficient) but what you write is also good, so I will change it to this.

Reply:
Changed, but I'd be interested in the parts that were deficient. I had in the interim that another pass is made for formatting. Should that be stated? And I think I added in the interim M-Expressions instead of Expressions. With that we can remove the WL part because it is not clear that we are 100% WL (we always be 100% Mathics though.)

Reply:
With WL I mean WL M-Expressions. However, it is true that in WMA the form of the resulting expression could be a little different from what we get in Mathics. 
Regarding the whole process, I guess that this deserves an entry in the documentation, and then we can add a link here to explain how a Plot expression is evaluated to a Graphics Expression, and then formatter to a Boxed Expression, to finally be rendered to a (for example) an SVG encoded picture.

Reply:
I agree the developer docs need going over.  In the meanwhile have adjusted the comment yet again go put back some of the stuff I mentioned in parallel with your comment.